packages/ember-metal-views/lib/htmlbars-renderer.js

```diff
@@ -84,6 +84,7 @@ Renderer.prototype.dispatchLifecycleHooks =
         case 'didUpdate': this.didUpdate(hook.view); break;
       }
 
+      ownerView._dispatching = 'didRender';
```

Comment:
Yes, it is initialized to `null` by `CoreView.prototype.init` [here](https://github.com/emberjs/ember.js/blob/e81233a6aa8cb29b72b92e799be510ad66046db7/packages/ember-views/lib/views/core_view.js#L47).